src/main/java/taxi/service/AuthenticationService.java

```diff
@@ -0,0 +1,9 @@
+package taxi.service;
+
+import java.util.Optional;
+import taxi.exception.AuthenticationException;
+import taxi.model.Driver;
+
+public interface AuthenticationService {
+    Optional<Driver> login(String login, String password) throws AuthenticationException;
```

Comment:
sorry, it was my mistake, findByLogin service method should return optional not login